digicampipe/io/event_stream.py

```diff
@@ -33,6 +31,15 @@ def event_stream(
             yield event
 
 
+def guess_source_from_path(path):
+    if path.endswith('.fits.fz'):
+        return zfits.zfits_event_source
+    elif path.endswith('.h5') or path.endswith('.hdf5'):
+        return hdf5.digicamtoy_event_source
+    else:
+        return hsm.hessio_event_source
+
+
```

Comment:
Just to comment on this I realized that now all `*event_source` stuff need to take the same arguments otherwise it breaks.

For instance we remove the `camera` argument in `zfits_event_source` the we also need to remove it from this function and from the others (digicamtoy, hessio). What if we just do: 

```python
event_stream(filelist, **kwargs)
```
then pass `**kwargs` to the `event_source` stuffs?

Reply:
sounds good. I'm doing it